README.md

```diff
@@ -24,281 +33,98 @@ of applications & services when it comes to rate limiting, authentication, autho
+instead it will opt to connect with what’s there and what’s being developed (think Envoy, Istio, GatewayAPI).
 
 Kuadrant is a system of cloud-native k8s components that grows as users’ needs grow.
-* From simple protection of a Service (via **AuthN**) that is used by teammates working on the same cluster, or “sibling” services, up to **AuthN** of users using OIDC plus custom policies.
+* From simple protection of a Service (via **AuthN**) that is used by teammates working on the same cluster, or “sibling” services, up to **AuthZ** of users using OIDC plus custom policies.
 * From no rate-limiting to rate-limiting for global service protection on to rate-limiting by users/plans
 
-towards a full system that is more analogous to current API Management systems where business rules
-and plans define protections and Business/User related Analytics are available.
+## Architecture
 
-## CustomResourceDefinitions
+Kuadrant relies on [Istio](https://istio.io/) and the [Gateway API](https://gateway-api.sigs.k8s.io/)
+to operate the cluster (istio's) ingress gateway to provide API management with **authentication** (authN),
+**authorization** (authZ) and **rate limiting** capabilities.
 
-A core feature of the kuadrant controller is to monitor the Kubernetes API server for changes to
-specific objects and ensure the owned k8s components configuration match these objects.
-The kuadrant controller acts on the following [CRDs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/):
+### Kuadrant components
 
 | CRD | Description |
 | --- | --- |
-| [RateLimitPolicy](apis/apim/v1alpha1/ratelimitpolicy_types.go) | Enable access control on workloads based on HTTP rate limiting |
-| [AuthPolicy](apis/apim/v1alpha1/authpolicy_types.go) | Enable AuthN and AuthZ based access control on workloads |
+| Control Plane | The control plane takes the customer desired configuration (declaratively as kubernetes custom resources) as input and ensures all components are configured to obey customer's desired behavior.<br> This repository contains the source code of the kuadrant control plane. |
+| [Kuadrant Operator](https://github.com/Kuadrant/kuadrant-operator) | A Kubernetes Operator to manage the lifecycle of the kuadrant deployment |
+| [Authorino](https://github.com/Kuadrant/authorino) | The AuthN/AuthZ enforcer. As the [external istio authorizer](https://istio.io/latest/docs/tasks/security/authorization/authz-custom/) ([envoy external authorization](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter) serving [gRPC](https://grpc.io/) service |
+| [Limitador](https://github.com/Kuadrant/limitador) | The external rate limiting service. It exposes a [gRPC](https://grpc.io/) service implementing the [Envoy Rate Limit protocol (v3)](https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/ratelimit/v3/rls.proto). |
```

Comment:
We could avoid the external link to https://grpc.io, IMO.

Reply:
link removed

Reply:
The other one in the Authorino description intentionally kept?

Reply:
fixed